src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,22 @@
-// Don't forget to import the React library
+import React from 'react';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import { PreparedTodo } from '../../types';
+import './TodoList.scss';
 
-// Create a `TodoList` component accepting an array of `preparedTodos` and
-// rendering them as a list
+interface Props {
+  preparedTodos: PreparedTodo[];
+}
 
-// Add a default export statement for TodoInfo component to use it in the other files
+export const TodoList: React.FC<Props> = ({ preparedTodos }) => {
+  return (
```

Comment:
use implicit returns when applicable
```suggestion
export const TodoList: React.FC<Props> = ({ preparedTodos }) => (
```